Assets/Scripts/Shop/ShopUIController.cs

```diff
@@ -104,18 +104,29 @@ private void CreateItemButton(Item.ItemType itemType)
 
     private void TryToBuyItem(Item.ItemType itemType)
     {
-        if (Item.IsItemPurchased(itemType))
-        {
-            SetAlertText("Item already purchased");
-            return;
-        }
+        // For now, letting them buy the same item multiple times
+        // if (Item.IsItemPurchased(itemType))
+        // {
+        //     SetAlertText("Item already purchased");
+        //     return;
+        // }
 
         int cost = Item.GetCost(itemType);
         if (PlayerBankAccount.GetCurrentBalance() >= cost)
         {
             PlayerBankAccount.SubtractHuskyDollars(cost);
             Item.PurchaseItem(itemType);
-            Inventory.AddItem(itemType);
+
+            int quantity = Item.GetQuantity(itemType);
+            if (PlayerPowerups.isMoAmmoActive)
+            {
+                quantity *= 2;
```

Comment:
Technically I think this would apply to grenades and powerups too but it's whatever